README.md

```diff
@@ -40,6 +40,12 @@ A synchronized element's content can also be based on concatenating multiple `<k
 
 To add new lines between items, use `'\n'` for example `sync:<key1> + '\n' + <key2>`.
 
+## Content wildcards
+
+A synchronized element's content can be chosen by pattern. For example
```

Comment:
Yeah... I don't totally understand how this plugin works with symbol layers, do you think there is functionality that should be added to work with symbols?